odoo/modules/loading.py

```diff
@@ -279,7 +279,7 @@ def load_module_graph(env, graph, status=None, perform_checks=True,
                 if not needs_update:
                     registry.setup_models(env.cr)
                 # Python tests
-                env['ir.http']._clear_routing_map()     # force routing map to be rebuilt
+                #env.registry.clear_cache('routing')     # force routing map to be rebuilt
```

Comment:
Wut?

Reply:
Before that, the routing map cache was not an ormcache an needed to be invalidated manually.

Since it is an ormcache, it feels like it is not useful anymore since the whole registry is modified, all ormcache are dropped.

I was not 100% sure of that and kept the line commented so that we can check that in details.